### PR TITLE
Use raw string literals

### PR DESCRIPTION
**Pull request by quantumsteve**
Please review the [developer documentation](https://github.com/QMCPACK/qmcpack/wiki/Development-workflow)
on the wiki of this project that contains help and requirements.

## Proposed changes

Describe what this PR changes and why.  If it closes an issue, link to it here
with [a supported keyword](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

Start changing our multi-line strings to C++11 raw string literals

This is part of #3675

## What type(s) of changes does this code introduce?
_Delete the items that do not apply_

- Code style update (formatting, renaming)
- Refactoring (no functional changes, no api changes)

### Does this introduce a breaking change?

- No

## Checklist

_Update the following with a yes where the items apply. If you're unsure about any of them, don't hesitate to ask.  This is
simply a reminder of what we are going to look for before merging your code._

- Yes. This PR is up to date with current the current state of 'develop'
- Yes. Code added or changed in the PR has been clang-formatted
- Yes/No. This PR adds tests to cover any new code, or to catch a bug that is being fixed
- Yes/No. Documentation has been added (if appropriate)

### CODE REVIEW

**Comment by ye-luo** [comment]
Could you also take care of test_2d_jastrow.cpp which is merged this morning?

**Review by PDoakORNL — DISMISSED** [review]
We've still got a bunch of bad formatting here, but this is a step forward.

**Comment by prckent** [comment]
Thanks Steve. Much improved.

File: src/QMCWaveFunctions/tests/test_pade_jastrow.cpp [hunk 1]
```diff
@@ -164,16 +164,16 @@ TEST_CASE("Pade2 Jastrow", "[wavefunction]")
...
-        <var id=\"J1H_A\" name=\"A\">0.8</var> \
-        <var id=\"J1H_B\" name=\"B\">5.0</var> \
-        <var id=\"J1H_C\" name=\"C\">-0.1</var> \
-    </correlation> \
-  </jastrow> \
-</wavefunction> \
-";
+  const char* jasxml = R"(<wavefunction name="psi0" target="e">
+  <jastrow name="J1" type="One-Body" function="pade2" print="yes" source="ion0">
+    <correlation elementType=\"H\">
```
Comment (ye-luo) [inline src/QMCWaveFunctions/tests/test_pade_jastrow.cpp]: bad line.

**Review by ye-luo — APPROVED** [review]
LGTM

**Review by ye-luo — APPROVED** [review]

**Comment by ye-luo** [comment]
Test this please

**Pull request merged**